features/CSV.feature

```diff
@@ -28,3 +28,26 @@ Feature: CSV Export
       toto,changmai,11,,[NOT_SCALAR]
 
       """
+
+  @mappings
+  Scenario: Exports a collection to CSV without specifying any field
+    Given an existing collection "nyc-open-data":"yellow-taxi"
+    And I "create" the following documents:
+      | _id               | body                                                       |
+      | "the-hive-th"     | { "city": "changmai", "district": 7, "unwanted": "field" } |
+      | "luca"            | { "city": "changmai", "district": [7, 8, 9] }              |
+      | "chuon-chuon-kim" | { "city": "hcmc", "district": 1, "nested": {"field": 1} }  |
+    And I refresh the collection
+    # collection:export
+    When I run the command "collection:export" with:
+      | arg  | nyc-open-data |     |
+      | arg  | yellow-taxi   |     |
+      | flag | --format      | csv |
+    Then I get the file in "nyc-open-data/yellow-taxi/documents.csv" containing
```

Comment:
This test does not prove that nested fields are correctly dumped, can you add an example of this?

Reply:
This is tested in the [previous feature](https://github.com/kuzzleio/kourou/pull/118/files/83de276ebb49c1c7eed81451a37317f6030edc8c..6f339cd90e25cc55470236e8ba510ad94dbba2bd#diff-81fb5b99c2065787b6b50d999406c43a52ca73efe2c7ee8cbf1e966d0b200f8dR27)

Reply:
oh yes, sorry :+1: